newIDE/app/src/ProjectsStorage/CloudStorageProvider/CloudProjectWriter.js

```diff
@@ -236,13 +236,15 @@ export const generateOnSaveProjectAs = (
 ) => {
   if (!saveAsLocation)
     throw new Error('A location was not chosen before saving as.');
-  const { name, gameId } = saveAsLocation;
+  const { name } = saveAsLocation;
   if (!name) throw new Error('A name was not chosen before saving as.');
   if (!authenticatedUser.authenticated) {
     return { wasSaved: false, fileMetadata: null };
   }
   options.onStartSaving();
 
+  const gameId = saveAsLocation.gameId || project.getProjectUuid();
```

Comment:
this change allows saving the projectUuid on first save of the cloud project, even if not registered.

Reply:
Make sense